Demo/src/main/java/com/braintreepayments/demo/Settings.java

```diff
@@ -27,6 +27,9 @@ public class Settings {
 
     static final String LOCAL_PAYMENTS_TOKENIZATION_KEY = "sandbox_f252zhq7_hh4cpc39zq4rgjcg";
 
+    private static final String XO_SANDBOX_TOKENIZATION_KEY = "sandbox_rz48bqvw_jcyycfw6f9j4nj9c";
+    private static final String XO_PRODUCTION_TOKENIZATION_KEY = "";
```

Comment:
Should this be empty?

Reply:
🤔 Probably not. I'll look into this one.

Reply:
I tested the default prod key, which shows the new checkout experience. I'll remove this empty key.

Reply:
Removed in https://github.com/braintree/braintree_android/pull/1024